### PR TITLE
Added Copy option for External Ip cell

### PR DESCRIPTION
**Pull request by dabasnidhi**
We show the copy button only on hover of external IP cell text
image

image

### CODE REVIEW

File: src/WebUI/Services/ServicesTable.tsx [hunk 1]
```diff
@@ -137,6 +139,29 @@ export class ServicesTable extends BaseComponent<IServicesComponentProperties> {
         return renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender);
     }
 
+    private static _renderExternalIpCell = (rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IServiceItem>, service: IServiceItem): JSX.Element => {
+        let textToRender = service.externalIP;
+        if (textToRender !== "-") {
+            const itemToRender = (
+                <div className="service-external-ip-cell">
+                    {textToRender}
+                    <Icon
```
[the remark this answers is not in the document]
Comment (dabasnidhi) [reply src/WebUI/Services/ServicesTable.tsx]: Moved as a common component to KubeCardWithTable

File: src/WebUI/Common/KubeCardWithTable.scss [hunk 2]
```diff
@@ -1,3 +1,9 @@
 .k8s-pods-status-count {
     padding-left: 8px;
+}
+
+.external-ip-cell .external-ip-copy-icon {
+    padding: 2px;
+    vertical-align: sub;
```
Comment (madhuv-msft) [inline src/WebUI/Common/KubeCardWithTable.scss]: what happens if i use flex-center?

File: src/WebUI/Common/KubeCardWithTable.tsx [hunk 3]
```diff
@@ -268,4 +270,36 @@ export function onPodsColumnClicked(
...
+        const itemToRender = (
+            <div
+                className="external-ip-cell"
+                onMouseOver={() => hoverHandler(rowIndex)}
+                onMouseLeave={() => hoverHandler(-1)}
+                onFocus={() => hoverHandler(rowIndex)}
+                onBlur={() => hoverHandler(-1)}
+                tabIndex={0}>
+                {textToRender}
+                {hoverRowIndex == rowIndex &&
```
Comment (madhuv-msft) [inline src/WebUI/Common/KubeCardWithTable.tsx]: ===

File: src/WebUI/Common/KubeCardWithTable.scss (hunk 2 above)
Comment (dabasnidhi) [reply src/WebUI/Common/KubeCardWithTable.scss]: Flex-center is already applied on these elements via parent table element. It does not align the items as needed

**Pull request merged**